CONTRIBUTING.MD

```diff
@@ -0,0 +1,64 @@
+Lyra Contributing Guidelines
+=======================
+
+New to Lyra? Check out "First PR" tag in the issues. These are issues have a lower level of effort for people looking to get their hands dirty in the Lyra codebase.
```

Comment:
Once we have a tag for this, we ought to markdown-link this up to the issue filter